src/main/java/ch/ivyteam/ivy/maven/engine/EngineClassLoaderFactory.java

```diff
@@ -77,16 +77,22 @@ public URLClassLoader createEngineClassLoader(File engineDirectory) throws IOExc
     List<File> osgiClasspath = getOsgiBootstrapClasspath(engineDirectory);
     addToClassPath(osgiClasspath, new File(engineDirectory, OsgiDir.PLUGINS),
             new WildcardFileFilter("org.eclipse.osgi_*.jar"));
-    osgiClasspath.add(0, maven.getJar("org.slf4j", "slf4j-api", SLF4J_VERSION));
-    osgiClasspath.add(0, maven.getJar("org.slf4j", "slf4j-simple", SLF4J_VERSION));
-    osgiClasspath.add(0, maven.getJar("org.slf4j", "log4j-over-slf4j", SLF4J_VERSION));
+    getSlf4jJars().forEach(slf4j -> osgiClasspath.add(0, slf4j));
     if (maven.log.isDebugEnabled())
```

Comment:
alternative: `osgiClasspath.addAll(0, getSlf4Jars());`

Reply:
thanks for the hint, wasn't aware on that API.